### ISSUE TITLE
Backgrid keyboard accessibility issues. (RM #3919)

### ISSUE THREAD

**Issue by dpage** [postgres/pgadmin4#2292]
***Issue migrated from Redmine: https://redmine.postgresql.org/issues/3919***
*Originally created by **Harshal Dhumal** at 2019-01-22 11:04:33 UTC.*

1. The Backgrid and sub-node should have accessibility including every button, cell and control.
2. The Backgrid table header is accessible through keyboard but it should be highlighted when it gets focus.

**Comment by dpage** [comment]
Issue closed on Redmine.
